src/witan/sen2/return/person_level/blade/csv.clj

```diff
@@ -71,15 +110,10 @@
 
 (def sen2-read-cfg
   "Configuration map for reading SEN2 module 0 \"SEN2\" CSV file into a dataset."
-  {:key-fn       sen2-csv-col-label->name
+  {:key-fn       #(or (sen2-src-col-name->col-name %) %)
```

Comment:
I used `or` rather than the optional `not-found` argument of [get](https://clojuredocs.org/search?q=get) to permit {"column name" nil} key-value pairs in the `src-col-name->col-name` maps to still read the column by it's source file name.

Although not used here, in some client applications I used nil values to specify columns to drop in `::post-fn` after reading from XLSX (`sheet-> dataset` doesn't support column allow|block lists).

Kept the construct in the standard SEN2 Blade CSV Export read here so code is as similar as possible to the bespoke code for XLSX reads.